kernel/kernel-defaults/src/test/resources/workload_specs/basic_append/table_info.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "basic_append",
```

Comment:
Can you also include: which engine created this table?
Can you also include: how did you create this table? e.g. let's say I fix a bug and now I want to re-create this table -- how did you create it in the first place?

We should at the very least include the SQL or pyspark etc code here, too. Just as JSON but it's a start.

This is the same problem DAT had :)